keylime/web/base/server.py

```
@@ -555,6 +555,10 @@ def _set_bind_interface(self, **kwargs: Any) -> None:
         """
 
         self._set_option("bind_interface", **kwargs)
+        if self.bind_interface == "*":
+            raise ValueError(
+                "bind address '*' is not supported; use '0.0.0.0' for IPv4 or '::' for IPv4+IPv6 dual-stack"
+            )
```

Ruff (0.15.21)

[warning] Avoid specifying long messages outside the exception class

(TRY003)